cirq-core/cirq/ops/gateset.py

```diff
@@ -167,6 +167,25 @@ def _value_equality_values_(self) -> Any:
             self._ignore_global_phase,
         )
 
+    def _json_dict_(self):
+        gate_json = self.gate
+        if isinstance(self.gate, type):
+            gate_json = protocols.json_cirq_type(self.gate)
```

Comment:
We can make a helper method `_gate_json`, similar to `_gate_str`, for `gate_json` use it in all _json_dict_ implementations to replace these 3 lines.

Reply:
Done. Only a couple of the GateFamilies use it, but I can imagine it will be useful for future subclasses.